src/index.ts

```diff
@@ -147,14 +142,11 @@ class SvgChunkWebpackPlugin {
 				compilation.emitAsset(output.filename, output.source);
 
 				sprites.push({ entryName, source: output.source });
-				spritesManifest[output.entryName] = output.manifestItem;
-				spritesList.push(output.spriteItem);
+				spritesManifest[output.entryName] = output.svgPaths;
 				spritesList.push({
-					name: entryName,
-					content: output.spriteItem,
-					svgs: svgsDependencies.map((moduleDependency: NormalModule) =>
-						path.basename(moduleDependency.userRequest, '.svg')
-					)
+					name: output.entryName,
+					content: output.source.source(),
```

Comment:
I don't recommend do it until you really need this, because getting source take a time

Reply:
Understood, this call was move to https://github.com/yoriiis/svg-chunk-webpack-plugin/blob/2f8c83bd81b74065481a1a7fe92f5eb3c332705d/src/preview.ts#L68

I could store the sprite value directly in the cache and access it instead of use the `RawSource`

Reply:
Done in fb1c242f7c290c6bddd4fc7143bae130680e8dae